src/dc_chat.rs

```diff
@@ -1001,7 +991,7 @@ unsafe fn set_draft_raw(context: &Context, chat_id: uint32_t, msg: *mut dc_msg_t
     // save new draft
     if !msg.is_null() {
         if (*msg).type_0 == Viewtype::Text {
-            if (*msg).text.is_null() || *(*msg).text.offset(0isize) as libc::c_int == 0i32 {
+            if (*msg).text.is_none() {
```

Comment:
You change the logic here. Existing code checks for
```
(*msg).text.is_none() || (*msg).text.unwrap().is_empty()
```

Reply:
@KAction If I remember correctly you added the check for empty string but removed it now?